CHANGELOG.unreleased.md

```diff
@@ -26,4 +26,6 @@
 > Keep this list alphabetically ordered to avoid merge conflicts
 
 <!--packages-start-->
+
+- xo-server: patch
```

Comment:
Respect the format otherwise `gen-deps-list.js` will not work :slightly_smiling_face: 

```suggestion
- xo-server patch

```

Reply:
We missed this :grimacing: